advisories/github-reviewed/2022/03/GHSA-3mpp-xfvh-qh37/GHSA-3mpp-xfvh-qh37.json

```diff
@@ -27,6 +27,38 @@
           ]
         }
       ]
+    },
+    {
+      "package": {
+        "ecosystem": "npm",
+        "name": "node-ipc"
+      },
+      "ranges": [
+        {
+          "type": "ECOSYSTEM",
+          "events": [
+            {
+              "introduced": "0"
```

Comment:
this seems too broad; node-ipc's problems are in 9.2.2 and two of the v10 versions and v11+, iirc